x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackClientPlugin.java

```diff
@@ -414,17 +413,15 @@ public List<ActionType<? extends ActionResponse>> getClientActions() {
         );
 
         // TSDB Downsampling / Rollup
-        if (IndexSettings.isTimeSeriesModeEnabled()) {
-            actions.add(RollupIndexerAction.INSTANCE);
-            actions.add(RollupAction.INSTANCE);
-        }
+        actions.add(RollupIndexerAction.INSTANCE);
```

Comment:
Maybe adding the rollup actions can move a few lines above in the `Arrays.asList()` statement like everything else.